ml/src/test/scala/ai/h2o/sparkling/ml/algos/BinomialPredictionTestSuite.scala

```diff
@@ -55,12 +55,13 @@ class BinomialPredictionTestSuite extends FunSuite with Matchers with SharedH2OT
     assert(!predictions.columns.contains("detailed_prediction"))
   }
 
-  test("detailedPredictionCol content with contributions") {
+  test("detailedPredictionCol content with contributions and assignments") {
     val algo = new H2OGBM()
       .setSplitRatio(0.8)
       .setSeed(1)
       .setWithDetailedPredictionCol(true)
       .setWithContributions(true)
+      .setLeafNodeAssignmentsEnabled(true)
```

Comment:
nit: WDYT about renaming property to `withLeafNodeAssignments`? Wouldn't it fit the naming conventions better? (`withContributions`, `WithDetailedPredictionCol`)

Reply:
Yup, agree